docs/data_collection.rst

```diff
@@ -0,0 +1,99 @@
+Data Collection
+===============
+
+This document describes how snippets data is collected, stored and used.
+
+Snippets follows Mozilla's established best practices for data collection
+and storage of personally identifiable information.
+
+Data Collection: Impressions
+----------------------------
+
+Immpression data is sampled at a rate of ~10%. For a tracked impression the following
+information is sent to the stats server.
+
+.. code-block:: json
+
+    snippet_name
```

Comment:
This would be better as a definition list describing what the value of each field is; we can't assume users understand what a locale or metric is, nor is it clear whose locale is being stored.